lib/cea/cea608_memory.js

```diff
@@ -360,88 +250,12 @@ shaka.cea.Cea608Memory = class {
   }
 };
 
-shaka.cea.Cea608Char = class {
```

Comment:
A very similar class was being used for CEA-708, so I broke it out into a utility.